### ISSUE TITLE
Details button on TimetableDetailScreen does not work

### ISSUE THREAD

**Issue by hueyy** [uclapi/ucl-assistant-app#21]
This button does not do anything at the moment.

<img src="https://user-images.githubusercontent.com/6523121/53209244-d1a41680-3631-11e9-9794-9a2d87f5f936.jpg" width="200" />

It should probably open the PersonDetailScreen with the profile of the lecturer / instructor / etc.

However, the data I'm working with is a bit different from that received from the /search/people endpoint:

```
"lecturer": {
    "name": "HIRSCH, Robin (Prof)",
    "department_name": "Computer Science",
    "department_id": "COMPS_ENG",
    "email": "ucacrdh@ucl.ac.uk"
}
```
 There are a few options:

1. We could remove the Details button altogether
2. We could open the PersonDetailScreen anyway and only show the data we have available (i.e. missing status and with a slightly different department name).
3. We could change the Details button to an Email button and open the email client upon press (as with the PersonDetailScreen)

I'm inclined towards Option 3 because this saves the end-user 1 tap. But Option 2 does seem reasonable as well.

**Comment by ChristopherHammond13** [comment]
There are a couple of options:
1) I could return the lecturer's UPI as part of the request and then a search can be performed based on that (as it's unique)
2) We could search for the professor using the search API and show the first result in the details screen
3) We could send the user to the search screen with the name pre-entered

Another option is showing the 'module leader' in the calendar, and letting them email to that address? As of the latest changes, there can be multiple bookers of a room (example: Computing labs that are booked both by the lecturer and the Admin Staff for the same date), but each module should have a leader we can show to users.

**Comment by hueyy** [comment]
REsolved by #24
